openpower/package/openpower-pnor-p10/openpower-pnor-p10.mk

```diff
@@ -3,7 +3,7 @@
 # openpower_pnor_p10
 #
 ################################################################################
-
+HCODE_STAGE_DIR = $(STAGING_DIR)/../../../build/hcode-p10-*opmst10/output/images
```

Comment:
The typical way of handling this sort of thing is for each package to deliver content into the main staging area, rather than having the op-build code contain package-specific paths.  That way the high-level op-build layer remains agnostic.  If you look at openpower/package/hcode-p10/hcode-p10.mk it already delivers the images into the $(STAGING_DIR)/hcode/ directory.  You should modify that file instead of this one.  Even better would be to create a layer of indirection where  hcode-p10.mk calls some kind of generic script inside the hcode src tree that copies all of the useful information out to the staging area.

Reply:
Okay